src/CarbonAware.WebApi/src/Program.cs

```diff
@@ -11,18 +11,18 @@
 
 builder.Services.AddControllers(options =>
 {
-    options.Filters.Add<HttpResponseExceptionFilter>();
+	options.Filters.Add<HttpResponseExceptionFilter>();
```

Comment:
nit: did the IDE change all space to tabs (in this file)?

Reply:
Fixed!